src/frontend-pwa/src/components/appNav/OfficeNavButton/OfficeNavButton.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * @summary Customizable Navigation button
+ *          Customizable Types:
+ *                -hex color
+ *                -Icons to display
+ *                -routes to route to
+ *                -disabled logic to implement
+ * @type {{
+ *          hex: string,
+ *          icon: string,
+ *          routes: string,
+ *          disabled: boolean
+ *        }}
```

Comment:
```suggestion
 * @type {(
 *          hex: string,
 *          icon: string,
 *          routes: string,
 *          disabled: boolean
 *        )}
```

Reply:
This should be defined elsewhere and then declared here.